packages/contracts-bedrock/contracts/deployment/SystemDictator.sol

```diff
@@ -440,6 +440,24 @@ contract SystemDictator is OwnableUpgradeable {
         finalized = true;
     }
 
+    /**
```

Comment:
Just wondering if we want to have the steps be public or internal. I have flip flopped on this decision multiple times, I don't feel like there is an obvious answer so I'm fine with leaving as is